tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -717,3 +718,471 @@ def test_reads_during_server_pod_restart(self):
         assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Step 8: Successfully ran IOs after server pod restart")
         LOGGER.info("ENDED: Test to verify READs during data pod restart.")
+
+    # pylint: disable=multiple-statements
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-44840")
+    def test_deletes_during_server_pod_restart(self):
+        """
+        This test tests DELETEs during server pod restart
+        """
+        LOGGER.info("STARTED: Test to verify DELETEs during server pod restart.")
+        LOGGER.info("STEP 1: Perform WRITEs/READs/Verify with variable object sizes")
+        users_org = self.mgnt_ops.create_account_users(nusers=1)
```

Comment:
seems one PR is merged which is creating IAM user in setup method, we can use the same if required but if we need 2 set of IAM users, we can create them here

Reply:
are we going to fix it here or any reply @rahul-hatwar

Reply:
Let do this fix in coming sprints